src/test/java/org/springframework/data/elasticsearch/core/SimpleElasticsearchDateMappingTests.java

```diff
@@ -37,6 +39,9 @@
 	@Test
 	public void testCorrectDateMappings() throws NoSuchFieldException, IntrospectionException, IOException {
 		XContentBuilder xContentBuilder = MappingBuilder.buildMapping(SampleDateMappingEntity.class, "mapping", "id", null);
```

Comment:
Should we rather move to a try-with-resources here?

Reply:
Not sure what the benefit of try-with-resource is in this case.  The close is needed to get the results put on the ByteArrayOutputStream, just like in MappingBuilderTest.  I just didn't pull out the 3 lines liek in MappingBuidlerTest because it was one test.  I debated creating a TestUtil class or the like, so they could share the method, but it didn't seem worth it for two tests.